api/src/lib/aws.ts

```diff
@@ -143,8 +143,7 @@ function getSQSClient() {
   return sqs
 }
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-async function sendSqsMessage(queueUrl: string, messageBody: any) {
+async function sendSqsMessage(queueUrl: string, messageBody: unknown) {
```

Comment:
This post makes a compelling argument for preferring `unknown` over `any` for typing values that are passed through to `JSON.stringify()`, so decided to go with it.